specification/security/resource-manager/Microsoft.Security/preview/2021-01-01-preview/examples/secureScoreControlDefinitions/ListSecureScoreControlDefinitions_managementGroup_example.json

```diff
@@ -0,0 +1,83 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01-preview",
+    "scope": "providers/Microsoft.Management/managementGroups/groupId"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "/providers/Microsoft.Security/SecureScoreControlDefinitions/a9909064-42b4-4d34-8143-275477afe18b",
```

Comment:
The ids in this file should start with the MG scope

Reply:
Currently, there are only tenant level definitions, that should be listed in all the tenant's resources.
There are no management group's specific definitions